<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMTCClass</class>
 <widget class="QWidget" name="NMTCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>网易云音乐缓存文件转换器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <widget class="QPushButton" name="pb_loadPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>载入目录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>9</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>540</width>
     <height>300</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>曲名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>大小</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>已转换</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="title_savepath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>保存位置:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="savePathEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="openPath">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>49</y>
     <width>30</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_convertAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>全部转换</string>
   </property>
  </widget>
  <widget class="QComboBox" name="convertAllMode">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>一般(单线程)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>快速(双线程)</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="ca_progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>430</width>
     <height>32</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>pathEdit</zorder>
  <zorder>pb_loadPath</zorder>
  <zorder>toolButton</zorder>
  <zorder>table</zorder>
  <zorder>title_savepath</zorder>
  <zorder>savePathEdit</zorder>
  <zorder>openPath</zorder>
  <zorder>pb_convertAll</zorder>
  <zorder>convertAllMode</zorder>
  <zorder>ca_progressBar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
